oak-doc/src/site/site.xml

```diff
@@ -37,8 +37,7 @@ under the License.
     </menu>
     <menu name="Main APIs">
       <item href="https://s.apache.org/jcr-2.0-spec/index.html" name="JCR API" />
-      <item href="https://jackrabbit.apache.org/jcr/jcr-api.html" name="Jackrabbit API" />
-      <item href="oak_api/overview.html" name="Oak API" />
+      <item href="oak_api/overview.html" name="Oak/Jackrabbit API" />
     </menu>
```

Comment:
Same here; dropping the reference to the Jackrabbit API page is ok, but claiming that the Oak page defines the "Jackrabbit API" is not.

Reply:
>  but claiming that the Oak page defines the "Jackrabbit API" is not.
I don't think I do that. It is just that the Jackrabbit API is now being maintained at Oak and if you look for the most recent documentation you find it there and no longer in "JR2". But feel free to propose something else.

Reply:
`jackrabbit-(oak)-api` is the artifact id and its name is "Jackrabbit API" (https://github.com/apache/jackrabbit-oak/blob/85c7f002219f0c8435e4bdacbb8fe56dcf5efe83/oak-jackrabbit-api/pom.xml#L30). I never heard of "Jackrabbit JCR API extensions" but I do agree that Jackrabbit exposes API apart from Jackrabbit API which makes it a bit confusing. BTW Oak also has a dedicated API bundle which is only part of the API exposed by Oak (https://github.com/apache/jackrabbit-oak/tree/trunk/oak-api).
I would strongly suggest to keep the original meaning of "Jackrabbit API" (i.e. the part which moved to Oak) and to use another name for the full API exposed by Jackrabbit.